03-setup-ingress-controller/nginx.md

```diff
@@ -272,17 +272,17 @@ Steps to follow:
 3. Then, create the [echo](assets/manifests/echo_deployment.yaml) and [quote](assets/manifests/quote_deployment.yaml) deployments:
```

Comment:
Manifest file path for the `echo` and `quote` deployments is incorrect. Should be:

```
[echo](assets/manifests/nginx/echo_deployment.yaml)
[quote](assets/manifests/nginx/quote_deployment.yaml)
```